opentelekomcloud/resource_opentelekomcloud_cce_node_pool_v3_test.go

```diff
@@ -0,0 +1,203 @@
+					resource.TestCheckResourceAttr(resourceName, "scale_down_cooldown_time", "100"),
+					resource.TestCheckResourceAttr(resourceName, "priority", "1"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckCCENodePoolDestroy(s *terraform.State) error {
+	config := testAccProvider.Meta().(*Config)
+	cceClient, err := config.cceV3Client(OS_REGION_NAME)
+	if err != nil {
+		return fmt.Errorf("Error creating Open Telekom Cloud CCE client: %s", err)
+	}
+
+	var clusterId string
+	var nodepollId string
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type == "opentelekomcloud_cce_cluster" {
+			clusterId = rs.Primary.ID
+		}
+
+		if rs.Type == "opentelekomcloud_cce_node_pool" {
+			nodepollId = rs.Primary.ID
+		}
+
+		if clusterId == "" || nodepollId == "" {
+			continue
+		}
+
+		_, err := nodepools.Get(cceClient, clusterId, nodepollId).Extract()
+		if err == nil {
+			return fmt.Errorf("Node still exists")
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckCCENodePoolExists(n string, cluster string, nodePool *nodepools.NodePool) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+		c, ok := s.RootModule().Resources[cluster]
+		if !ok {
+			return fmt.Errorf("Cluster not found: %s", c)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No ID is set")
+		}
+		if c.Primary.ID == "" {
+			return fmt.Errorf("Cluster id is not set")
+		}
+
+		config := testAccProvider.Meta().(*Config)
+		cceClient, err := config.cceV3Client(OS_REGION_NAME)
+		if err != nil {
+			return fmt.Errorf("Error creating Open Telekom Cloud CCE client: %s", err)
+		}
+
+		found, err := nodepools.Get(cceClient, c.Primary.ID, rs.Primary.ID).Extract()
+		if err != nil {
+			return err
+		}
+
+		if found.Metadata.Id != rs.Primary.ID {
+			return fmt.Errorf("Node Pool not found")
+		}
+
+		*nodePool = *found
+
+		return nil
+	}
+}
+
+func testAccCCENodePool_Base(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+data "opentelekomcloud_availability_zones_v3" "test" {}
+
+resource "opentelekomcloud_compute_keypair" "test" {
+  name = "%s"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAjpC1hwiOCCmKEWxJ4qzTTsJbKzndLo1BCz5PcwtUnflmU+gHJtWMZKpuEGVi29h0A/+ydKek1O18k10Ff+4tyFjiHDQAT9+OfgWf7+b1yK+qDip3X1C0UPMbwHlTfSGWLGZquwhvEFx9k3h/M+VtMvwR1lJ9LUyTAImnNjWG7TAIPmui30HvM2UiFEmqkr4ijq45MyX2+fLIePLRIFuu1p4whjHAQYufqyno3BS48icQb4p6iVEZPo4AE2o9oIyQvj2mx4dk5Y8CgSETOZTYDOR3rU2fZTRDRgPJDH9FWvQjF5tA0p3d9CoWWd2s6GKKbfoUIi8R/Db1BSPJwkqB jrp-hp-pc"
+}
+
+resource "opentelekomcloud_cce_cluster_v3" "test" {
+  name                   = "%s"
+  cluster_type           = "VirtualMachine"
+  flavor_id              = "cce.s1.small"
+  vpc_id                 = opentelekomcloud_vpc_v1.test.id
+  subnet_id              = opentelekomcloud_vpc_subnet_v1.test.id
+  container_network_type = "overlay_l2"
+}
+`, TestAccCCEClusterV3_basic(rName), rName, rName)
```

Comment:
`TestAccCCEClusterV3_basic` is a test method. What are you trying to pass to the format expression?